pkg/pillar/types/zedroutertypes.go

```diff
@@ -756,7 +855,7 @@ func (config *NetworkInstanceConfig) IsIPv6() bool {
 func (config *NetworkInstanceConfig) WithUplinkProbing() bool {
 	switch config.Type {
 	case NetworkInstanceTypeLocal:
-		return IsSharedPortLabel(config.PortLogicalLabel)
+		return IsEveDefinedPortLabel(config.PortLogicalLabel)
```

Comment:
This assumes that we will never define a well-known label for something which does not include a management interface.

So it might make sense to have the explicit case here as well even if all current well-known labels would be included?

Reply:
So this is going to change in the next PR. This function will not be named `WithUplinkProbing` because "Uplink" here is indeed confusing. For example, the new EVE-defined label "all" includes app-shared interfaces.
To avoid too large PR I wanted to submit portprober component separately. Still I had to bring in some changes from the `pillar/types` packages, but only those that do not change/break anything. The next PR will be the main one that will bring in all the changes to zedrouter for this feature.